### ISSUE TITLE
TopArticles API should have an option to exclude satire sources

### ISSUE THREAD

**Issue by glciampaglia** [IUNetSci/hoaxy-backend#7]
Per title. We would like to add a parameter to the TopArticles API endpoint, e.g. `exclude=['satire',]`.

**Comment by filmenczer** [comment]
Thank you @shaochengcheng !  

This list of 'satire' sources is much longer than we expected... Many sites that spread fake news are labeled satire, so we would probably not want to exclude these in the current version of the Hoaxy front-end. But it is a good feature to have, especially if we can get more reliable tags in the future.

For the Mashape API docs, can you write here how this works in your words (is there an optional param 'exclude' that takes a comma-separated list of keywords from a list of available keywords?). Then I can suggest rewording if needed. And how would a user know which keywords are available? Should we only use tags from opensources?

As soon as we take care of the documentation, we can close this issue.

**Comment by shaochengcheng** [comment]
@filmenczer, I already put a draft version of API docs in the Mashape, please help me reword it.

https://market.mashape.com/truthy/hoaxy#toparticles

Thanks
Chengcheng

**Comment by shaochengcheng** [comment]
@filmenczer, I already put a draft version of API docs in the Mashape, please help me reword it.

https://market.mashape.com/truthy/hoaxy#toparticles

Thanks
Chengcheng

**Comment by filmenczer** [comment]
Great @shaochengcheng !! I updated the description. Could you make it so the default is empty (no exclusion)? That is how I have in the description now. This way the default behavior will not change. Thanks!

**Comment by shaochengcheng** [comment]
Yes, of course. I altered the code to set `[]` as the default (no filtering). Please tell @benabus (if his responsibility) to adjust the API query parameters if we would like to exclude the satire sites on Hoaxy frontend.
